decidim-proposals/spec/commands/decidim/proposals/vote_proposal_spec.rb

```diff
@@ -10,7 +10,13 @@ module Proposals
         let(:current_user) { create(:user, organization: proposal.feature.organization) }
         let(:command) { described_class.new(proposal, current_user) }
 
-        describe "when the vote is not valid" do
+        context "in normal conditions" do
```

Comment:
Isn't this already handled in the `"when the vote is valid"` context?

Reply:
Yes! I've removed the redundancy (but this doesn't show in here because this line wasn't modified)